src/libs/migrations/RenameActiveClientsKey.js

```diff
@@ -0,0 +1,35 @@
+import Onyx from 'react-native-onyx';
+import _ from 'underscore';
+import ONYXKEYS from '../../ONYXKEYS';
+
+// This migration changes the name of the Onyx key ACTIVE_CLIENTS from activeClients2 to activeClients
+export default function () {
+    return new Promise((resolve) => {
+        // Connect to the old key in Onyx to get the old value of activeClients2
+        // then set the new key activeClients to hold the old data
+        // finally remove the old key by setting the value to null
+        const connectionID = Onyx.connect({
```

Comment:
NAB, we could maybe avoid this `Onyx.connect()` dance by just making the migrations internal to Onyx that run before `init()`. I would think most of them will be grabbing data, renaming keys, etc.